src/components/AppShell/style.js

```diff
@@ -12,8 +12,9 @@ const flexRow = css`
 
 export const AppShellStyled = styled.div`
   ${flexColumn}
-  min-height: 100%;
+  height: 100vh;
```

Comment:
@ssvmvss let me know if this doesn't work for you but `min-height:100%` wasn't working inside of Reply! Happy to try out some other things if that's the case!